### PR TITLE
Updates for Linux compatibility

### PR DESCRIPTION
**Pull request by ShelleyPeterson**
changed path directories and system commands to allow for use in both Windows and Linux

### CODE REVIEW

File: WADE.R [hunk 1]
```diff
@@ -71,13 +71,13 @@ ui <- fluidPage(
...
                        choices = list("AMR Profile" = "AMR_ALL",
-                                      #"AMR Alleles" = "AMR",
+                                      "AMR Alleles" = "AMR",
                                       "23S rRNA Alleles" = "rRNA23S",
                                       "MLST" = "MLST",
                                       "NG-STAR" = "NGSTAR",
                                       "NG-MAST" = "NGMAST",
                                       "All Routine Analyses" = "ALL",
-                                      "MasterBlastR" = "MASTERBLASTR",
+                                      #"MasterBlastR" = "MASTERBLASTR",
```
Comment (apetkau) [inline WADE.R:80]: Was commenting this out a part of what was needed for Linux compatibility or was it for a different change?

**Review by apetkau — COMMENTED** [review]
This looks great @ShelleyPeterson. Amazing job 😄. I did have one comment listed below.

Additionally, another possible option to create platform-independent paths in R is to use `file.path`: https://rdrr.io/r/base/file.path.html. I don't think it's necessary to change any of the code you have now though. I only wanted to mention this for your information.

File: WADE.R (hunk 1 above)
Comment (ShelleyPeterson) [reply WADE.R:80]: This is a function we just use for research so I took it out of the github version

Comment (apetkau) [reply WADE.R:80]: Ah, okay, makes sense. I think everything looks good then 😄

**Review by apetkau — APPROVED** [review]
This all looks great. You can merge it in @ShelleyPeterson 😄

**Pull request merged**